doc/classes/TreeItem.xml

```diff
@@ -226,11 +226,19 @@
 				Returns the next sibling TreeItem in the tree or a null object if there is none.
 			</description>
 		</method>
+		<method name="get_next_in_tree">
+			<return type="TreeItem" />
+			<param index="0" name="wrap" type="bool" default="false" />
+			<description>
+				Returns the next TreeItem in the tree (in the context of a Depth-first search) or a null object if there is none.
+				If [param wrap] is enabled, the method will wrap around to the first visible element in the tree when called on the last element, otherwise it returns [code]null[/code].
```

Comment:
"first visible" does not apply here, no?

Reply:
This was a copy-paste mistake.